wcomponents-theme/src/main/js/wc/ui/imageEdit.js

```diff
@@ -449,8 +449,10 @@ function(has, mixin, wcconfig, Widget, event, classList, timers, prompt, i18n, f
 			});
 
 			fbCanvas._activeObject = null;
-
-			fbCanvas.setActiveGroup(group.setCoords()).renderAll();
+			// remove all objects and re-render
+			fbCanvas.clear().renderAll();
```

Comment:
Wow that's what fabric makes you do to select all objects now? You have to remove and add? Are you sure?

This looks pretty recent http://jsfiddle.net/softvar/mRA8Z/

Still, if that's what you gotta do then that's what you gotta do. I wouldn't know, it's a FabricJS thing.

Reply:
You inspired me to take another look at this... and I found this doco from fabric about the breaking API http://fabricjs.com/v2-breaking-changes-2. I will make a change to use the ActiveSelection.